src/storm-pomdp/parser/PomdpSolveParser.cpp

```diff
@@ -0,0 +1,422 @@
+#include "PomdpSolveParser.h"
+#include "storm/adapters/RationalNumberAdapter.h"
+#include "storm/api/builder.h"
+#include "Utility.h"
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <vector>
+#include <map>
+#include <unordered_set>
+#include <iostream>
+
+namespace storm {
+namespace pomdp {
+namespace parser {
+
+struct POMDP {
```

Comment:
You should not call this simply ``POMDP``, maybe something like ``POMDPcomponents``?